src/neptune/common/utils.py

```diff
@@ -268,3 +268,9 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+
+def is_unsupported_float(value: float) -> bool:
```

Comment:
`src/neptune/common` means that it's a part of both `neptune.legacy` and regular `neptune` packages. I don't think there are any usage of it in legacy client. Let's move it to something like `src/neptune/internal/types/utils.py`

Reply:
Moved